lms/djangoapps/grades/transformer.py

```diff
@@ -49,6 +50,12 @@ def collect(cls, block_structure):
         """
         block_structure.request_xblock_fields(*cls.FIELDS_TO_COLLECT)
         cls._collect_max_scores(block_structure)
+        collect_unioned_set_field(
+            block_structure=block_structure,
+            transformer=cls,
+            merged_field_name='containing_subsections',
```

Comment:
`containing_subsections` doesn't seem quite as accurate, now that a block's `TBD` value is a set that can contain itself. Any other suggestions? `subsection_grades_to_be_updated_on_block_score_change` is more accurate, but a bit wordy.